docs/data/data-grid/pagination/pagination.md

```diff
@@ -120,6 +120,11 @@ React.useEffect(() => {
 <DataGrid rowCount={rowCountState} />;
 ```
 
+:::warning
+The `rowCount` prop is used in server-side pagination mode to inform the DataGrid about the total number of rows in your dataset.
+This prop is ignored when the `paginationMode` is set to `client`, meaning the pagination is handled on the client-side.
```

Comment:
```suggestion
The `rowCount` prop is used in server-side pagination mode to inform the DataGrid about the total number of rows in your dataset.
This prop is ignored when the `paginationMode` is set to `client`, i.e. when the pagination is handled on the client-side.
```

Reply:
No i.e. per Vale rule

Reply:
Hmm... I wonder what could be the reason, since I feel such abbreviations (like e.g. which seems to be [allowed](https://github.com/mui/mui-x/blob/next/.github/styles/Blog/Typos.yml) are fairly understood and could be helpful to save some space.

Reply:
`vale sync && vale docs/data/data-grid/pagination/pagination.md` will error